source/code/plugins/oms_common.rb

```diff
@@ -478,6 +480,141 @@ class Common
     @@tzRightFolder = 'right/'
 
     class << self
+
+      # Internal methods
+      # (left public for easy testing, though protected may be better later)
+
+      def clean_hostname_string(hnBuffer)
+        s = hnBuffer.strip
+        # Take off any port numbers:
+        if /\A(\h{4}:\h{4}:\h{4}:\h{4}:\h{4}:\h{4}:\h{4}:\h{4}):\d+\z/ =~ s
+            return $1
+        elsif not is_like_ipv6_string?(hnBuffer) and /\A(.*):\d+\z/ =~ s
+            return $1
+        end
+        return s
+      end
+
+      def has_Designated_HostnameFile?
+        return false if @@HostnameFilePath.nil?
+        return false if @@HostnameFilePath.length < 8 # 8 is first Guess.  Need policy here.
```

Comment:
Please explain the need for this line. We currently don't have a policy safeguarding against our own file path strings.

Reply:
Semantics dictates it gives good information.  If that is a bad spec, I just need the good one.  I suspect it would be good to check non-zero string at least.

Reply:
Sure, checking that the string has a length > 0 would be fine.